wireapp/wire-android
# feat: message details reactions

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

[AR-2681 - UI for MessageDetailsScreen](https://wearezeta.atlassian.net/browse/AR-2681)

### Issues

Not displaying reactions on message details.

### Causes (Optional)

Didn't exist.

### Solutions

Create `MessageDetailsScreen` and show reactions for selected message.

### Testing

#### How to Test

- Open App
- Open Conversation
- Long Click on a message with reactions
- Select Message Details
- Should see reactions for selected message


### Attachments (Optional)
<img src="https://user-images.githubusercontent.com/5890660/202419729-fdee01a9-da93-4f88-8721-0c7209be802a.png" alt="message details reactions" width="200" height="400" />
